src/components/helmetComponent/index.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import HelmetComponent from './index'
+
+describe('<HelmetComponent/>', () => {
+  let wrapper
+
+  describe('Props', () => {
+    beforeAll(() => (wrapper = shallow(<HelmetComponent title="test" />)))
```

Comment:
For these single-render tests, they should be in a separate `describe` block. As currently written, this suite relies on the tests running in exactly the order they appear in the file, which is not guaranteed. So, we'd need to have one `describe` with the single-render tests (i.e. **Snapshot** and **has correct title prop**) and a separate `describe` block for the remaining two. In addition, these don't need to be the same reallocated `wrapper` and instead could have the assignment of `const wrapper = ...` inside the test itself as it only belongs to this scope.